src/overlays/actors/ovl_Obj_Mure2/z_obj_mure2.c

```diff
@@ -25,33 +41,198 @@ const ActorInit Obj_Mure2_InitVars = {
     (ActorFunc)ObjMure2_Update,
     NULL,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure2/func_80B99F40.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure2/func_80B9A058.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure2/func_80B9A158.s")
+static f32 sDistSquared1[] = { SQ(1600.0f), SQ(1600.0f), SQ(1600.0f) };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure2/func_80B9A260.s")
+static f32 sDistSquared2[] = { SQ(1705.0f), SQ(1705.0f), SQ(1705.0f) };
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Obj_Mure2/func_80B9A2B0.s")
+#ifdef NON_MATCHING
+static s16 D_80B9A818[] = { 9, 12, 8 };
```

Comment:
if your intention is to put this inside the func when its matched, then you can put it inside now

Reply:
I can't since its used between multiple functions.